### PR TITLE
Register phub addon because of jq on SLE12-SP5

### PR DESCRIPTION
**Pull request by pdostal**
* Follows: #15075 
- Related ticket: [poo#112541](https://progress.opensuse.org/issues/112541)
- Verification run: http://pdostal-server.suse.cz/tests/102

### CODE REVIEW

File: tests/containers/bci_prepare.pm [hunk 1]
```diff
@@ -41,6 +41,8 @@ sub packages_to_install {
         my $version = "$version.$sp";
         push @packages, 'python3-devel';
         if ($version eq "12.5") {
+            # PackageHub is needed for jq
+            script_retry("SUSEConnect -p PackageHub/12.5/$arch", delay => 60, retry => 3);
```
Comment (grisu48) [inline tests/containers/bci_prepare.pm:45]: Why not use the provided standard subroutines?

```suggestion
            add_suseconnect_product(get_addon_fullname('phub'));
```

File: tests/containers/bci_prepare.pm [hunk 2]
```diff
@@ -41,6 +41,8 @@ sub packages_to_install {
         my $version = "$version.$sp";
         push @packages, 'python3-devel';
         if ($version eq "12.5") {
```
Comment (grisu48) [inline tests/containers/bci_prepare.pm:43]: What about 12-SP4? Pretty sure if `jq` is missing on 12-SP5 then 12-SP4 (and 12-SP3) will also complain.

Comment (jlausuch) [reply tests/containers/bci_prepare.pm:43]: yeah, I had to check myself till which version we are using in the tests :)

File: tests/containers/bci_prepare.pm (hunk 1 above)
Comment (pdostal) [reply tests/containers/bci_prepare.pm:45]: Open the VR and see the `BUILD` and `VERSION`.

File: tests/containers/bci_prepare.pm (hunk 2 above)
Comment (pdostal) [reply tests/containers/bci_prepare.pm:43]: We don't run this on 12-SP4. Or do we?

File: tests/containers/bci_prepare.pm (hunk 1 above)
Comment (grisu48) [reply tests/containers/bci_prepare.pm:45]: I see. Don't mind me resolving the issue here as well.

**Review by grisu48 — APPROVED** [review]

**Pull request merged**